### DISCUSSION TITLE
Question: Does using a version make sense when the repo is a configuration holder?

### DISCUSSION THREAD

**Discussion by aitorpazos** [olivierlacan/keep-a-changelog#505] [Q&A]
Hi all,
I am having a hard time convincing myself that using versions for repositories that hold configuration in a GitOps environment makes sense.

I am totally on-board of keeping a Changelog when changing configuration as well as it provide context and additional information to peers and myself from the future.

When software is published to a repository the concept of a version is clear. However, what does a version mean when every change is eventually promoted to prod? (given it passes the required validations)

Would it be acceptable to add my changelog entries like this:

```
## 2020-07-30

### Added

- Something was added

### Changed

- Something changed

## 2020-07-20

### Fixed

- Something was fixed
```

**Comment by andymadge** [comment]
I think the version numbering scheme you use is entirely separate from how you maintain the changelog. The changelog should be about recording what changes are in each release, but it doesn't specify any specific version numbering scheme to use.

The examples use numeric or explicitly [Semver-based](https://semver.org/) version numbers, but those aren't requirements.  Date-based version numbering as in your example is a perfectly valid versioning scheme. There is even a spec for it - https://calver.org/ (albeit a fairly flexible one)
